### PR TITLE
JsonLD VC BBS Go

### PR DESCRIPTION
**Pull request by iliecirciumaru**
Golang implementation of W3C JSON-LD BBS credentials

### CODE REVIEW

File: labs/jsonld-vc-bbs-go.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+parent: Labs
+---
+# Lab Name
+JsonLD VC BBS Go
+
+# Short Description
+Golang implementation of W3C JSON-LD BBS credentials
+
+# Scope of Lab
+This library fits into The Linux Foundation Decentralized-Trust branch. It implements issuance, verification, proof generation of [W3C JSON-LD](https://decentralized-id.com/web-standards/w3c/verifiable-credentials/data-integrity-bbs+/) credentials in Golang. The initially supported standards are  `BbsBlsSignature2020` and `BbsBlsSignatureProof2020`. Later we will add new standards as they are adopted.
```
Comment (tkuhrt) [inline labs/jsonld-vc-bbs-go.md:13]:
```suggestion
This library implements issuance, verification, proof generation of [W3C JSON-LD ZKP with BBS+ Verifiable Credentials](https://decentralized-id.com/web-standards/w3c/verifiable-credentials/data-integrity-bbs+/) in Golang. The initially supported standards are `BbsBlsSignature2020` and `BbsBlsSignatureProof2020`. Later we will add new standards as they are adopted.
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]
@denyeart @mbrandenburger : Can you confirm your sponsorship? Also, is this something that is already work that has been done in one of the Aries sub-projects?

**Comment by iliecirciumaru** [comment]
> @denyeart @mbrandenburger : Can you confirm your sponsorship? Also, is this something that is already work that has been done in one of the Aries sub-projects?

The functionality planned for this repo is partially available in the `https://github.com/hyperledger-archives/aries-framework-go`, but this repository is archived. I plan new repo to be very focused, like analogical libraries for Verifiable Credentials available in the JavaScript/TypeScript

**Comment by denyeart** [comment]
I agree to be a sponsor!

**Comment by mbrandenburger** [comment]
I support this proposal. While I think labs is a good place for this project, I am wondering if it could also be project of aries? @swcurran @ale-linux WDYT?

**Comment by swcurran** [comment]
There is the https://github.com/hyperledger/aries-bbs-go that was extracted from the Aries Framework Go after it was archived.  @ale-linux knows a lot more about that.  There may be synergies between the two.

**Comment by iliecirciumaru** [comment]
> There is the https://github.com/hyperledger/aries-bbs-go that was extracted from the Aries Framework Go after it was archived. @ale-linux knows a lot more about that. There may be synergies between the two.

Correct, this library will use `https://github.com/hyperledger/aries-bbs-go` for crypto-operations. Repository code will concern mainly operations with JSON-LD Verifiable Credential, but not crypto itself

**Comment by swcurran** [comment]
That makes sense — I wondered the level of the implementation.  Are you doing a W3C [VC-DI-BBS](https://github.com/w3c/vc-di-bbs) implementation?

**Comment by iliecirciumaru** [comment]
> That makes sense — I wondered the level of the implementation. Are you doing a W3C [VC-DI-BBS](https://github.com/w3c/vc-di-bbs) implementation?

I want to support eventually two standards:
- [old standard 2020](https://w3c-ccg.github.io/ld-cryptosuite-registry/#bbs-signature-2020)
- [new standard 2023](https://www.w3.org/TR/vc-di-bbs/)

Reasons to support old standard:
1. The code already exists, so I need only to publish it
2. Another open source libraries depend on it

File: labs/jsonld-vc-bbs-go.md (hunk 1 above)
Comment (iliecirciumaru) [reply labs/jsonld-vc-bbs-go.md]: Changed the description

**Review by vipinsun — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]

**Pull request merged**